core/src/main/java/org/keycloak/jose/jwk/JWKBuilder.java

```diff
@@ -17,15 +17,27 @@
 
 package org.keycloak.jose.jwk;
 
-import org.keycloak.crypto.KeyUse;
-
+import java.math.BigInteger;
 import java.security.Key;
+import java.security.interfaces.EdECPublicKey;
+import java.security.spec.EdECPoint;
+import java.util.Arrays;
+import java.util.Optional;
+
+import org.keycloak.common.util.Base64Url;
+import org.keycloak.common.util.KeyUtils;
+import org.keycloak.crypto.Algorithm;
+import org.keycloak.crypto.KeyType;
+import org.keycloak.crypto.KeyUse;
```

Comment:
It does not belong here. 

It seems some changes slipped here unintentionally. @jonkoops Correct?

Reply:
These are actually intentional changes, this code was previously in a separate directory that was activated by [a profile](https://github.com/keycloak/keycloak/blob/main/core/pom.xml#L78-L115) when the JDK version was 16 or higher. So essentially depending on the JDK version that you are using during build you would end up with the code from `src/main/java16` or `src/main/java`.

This change moves the code previously in the `src/main/java16` directory directly into `src/main/java`, as we can now always assume that we're compiling for JDK 17 or higher.

Reply:
@jonkoops Interesting. Ok, that works for me.